frontend/src/components/SkeletonSignUpPage.js

```diff
@@ -11,7 +11,7 @@ import {
 } from "@mui/material";
```

Comment:
If don't need this file can just delete? @hiongkaihan @TheSpaceCuber

Reply:
Fixed